extensions/swagger-ui/runtime/src/main/java/io/quarkus/swaggerui/runtime/SwaggerUiServletExtension.java

```diff
@@ -47,10 +47,9 @@ public void setResourceDir(String resourceDir) {
     @Override
     public void handleDeployment(DeploymentInfo deploymentInfo, ServletContext servletContext) {
         ResourceManager resourceManager = new FileResourceManager(new File(resourceDir));
-        //        ResourceManager resourceManager = new ClassPathResourceManager(servletContext.getClassLoader(),
-        //                SWAGGER_UI_RESOURCES_DIR);
         deploymentInfo.addOuterHandlerChainWrapper(
                 (handler) -> Handlers.path(handler).addPrefixPath(path, resource(resourceManager)));
+        log.info("Swagger UI served at " + path);
```

Comment:
I don't think we want that one either. At least, not at info level. IIRC, you had a debug level message similar to this one somewhere?

Reply:
Normally, I removed the other logs. I only keep this one, for me it makes sense to display this info to remember the default path/customized path.
```
11:29:20,129 INFO  [io.qua.dep.QuarkusAugmentor] Beginning quarkus augmentation
11:29:20,337 INFO  [org.jbo.threads] JBoss Threads version 3.0.0.Alpha4
11:29:20,473 WARN  [io.qua.config] Unrecognized configuration key "quarkus.undertow.resources" provided
11:29:20,766 WARN  [io.qua.swa.dep.SwaggerUiProcessor] Here
11:29:21,002 INFO  [io.qua.dep.QuarkusAugmentor] Quarkus augmentation completed in 873ms
11:29:21,031 INFO  [io.sma.ope.api.OpenApiDocument] OpenAPI document initialized: io.smallrye.openapi.api.models.OpenAPIImpl@4a003cbe
11:29:21,078 INFO  [io.qua.swa.run.SwaggerUiServletExtension] Swagger UI accessible at /swagger-ui
11:29:21,246 INFO  [org.xnio] XNIO version 3.7.0.Final
11:29:21,263 INFO  [org.xni.nio] XNIO NIO Implementation Version 3.7.0.Final
11:29:21,368 INFO  [io.quarkus] Quarkus 999-SNAPSHOT started in 1.371s. Listening on: http://[::]:8080
11:29:21,368 INFO  [io.quarkus] Installed features: [cdi, resteasy, smallrye-openapi, swagger-ui]
```